lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2676,6 +2687,26 @@ public function get_styles_block_nodes() {
 		return static::get_block_nodes( $this->theme_json );
 	}
 
+	/**
+	 * A public helper to get all styles nodes path in theme.json,
```

Comment:
```suggestion
	 * A public helper to get all style node paths in theme.json,
```

Nit: This is a bit awkwardly named or worded. Can we tweak it?